tests: add a list with all the MAUT tests we can currently pass

rmelotte opened the pull request:
Previously this list was contained in variables on Gitlab.com.

However, Gitlab is not the only place where we want to run the MAUT
tests we can currently pass.

Also, having this list in the repository makes it easier for people to
add new passing tests when working on a flow.

The list of passing controller tests is not added because we can
already pass them all, so all_controller_tests.txt can be used for the
same purpose.

Signed-off-by: Raphaël Mélotte <raphael.melotte@mind.be>

rmelotte committed:
tests: add a list with all the MAUT tests we can currently pass

Previously this list was contained in variables on Gitlab.com.

However, Gitlab is not the only place where we want to run the MAUT
tests we can currently pass.

Also, having this list in the repository makes it easier for people to
add new passing tests when working on a flow.

The list of passing controller tests is not added because we can
already pass them all, so all_controller_tests.txt can be used for the
same purpose.

Signed-off-by: Raphaël Mélotte <raphael.melotte@mind.be>

arnout reviewed (approved):
The list may already be out of date, but we can fix that later :-)

arnout commented:
I guess as soon as this is merged (i.e., now), the gitlab variable will be updated?

Also, did you document this anywhere?

rmelotte commented:
I updated [the wiki page](https://github.com/prplfoundation/prplMesh/wiki/Testing-in-CI) and made a hotfix to mention it in CONTRIBUTING.md: https://github.com/prplfoundation/prplMesh/pull/1171 .

I updated the variables on Gitlab.com already since this should get merged really soon by mergify.
